public/app/features/alerting/unified/triage/types.ts

```diff
@@ -0,0 +1,22 @@
+export type Domain = [Date, Date];
+export type Filter = [key: string, operator: '=' | '=!', value: string];
+
+export type WorkbenchRow = GenericGroupedRow | AlertRuleRow;
```

Comment:
Been thinking we should probably have a specific "type" key so we can make it easier to do union discrimination – wdyt?